docs/rfcs/1000-shamir-based-recovery.md

```diff
@@ -730,6 +731,121 @@ The claimer gets access to `reveal_token` and `data_key`, it can then retrieve `
 Then `ciphered_data` can be decrypted with `data_key`. From then on, the recovery works
 just like the recovery device system (see `parsec core import_recovery_device` CLI).
 
+## 4 - Delete a shamir setup
+
+Authenticated API:
+
+```json5
+[
+    {
+        "major_versions": [
+            4
+        ],
+        "req": {
+            "cmd": "shamir_recovery_deletion",
+            "fields": [
+                {
+                    "name": "certificate",
+                    // Contains a ShamirRecoveryDeletionCertificate
+                    "type": "bytes"
+                }
+            ]
+        },
+        "reps": [
+            {
+                "status": "ok"
+            },
+            {
+                // Cannot deserialize data into the expected certificate
+                "status": "invalid_data"
+            },
+            {
+                // Given timestamp does not match previous timestamps
+                "status": "incoherent_previous_timestamp"
```

Comment:
We tend to use the synonym `inconsistent` in the code base:
```
± git grep inconsistent | wc -l
15
[vinmic:~/repos/parsec-cloud-v3] [base] master+*
± git grep incoherent | wc -l  
4
```

Reply:
The 4 occurrences of `incoherent` were from your previous PR, sorry for not catching that earlier.

Reply:
Nice catch :fishing_pole_and_fish: 

So here you have the correction #7621